test/e2e/deploy/deploy.go

```diff
@@ -581,7 +581,7 @@ func EnsureDeployment(deploymentName string) *Deployment {
 				framework.Logf("reusing existing %s PMEM-CSI components", deployment.Name)
 				// Do some sanity checks on the running deployment before the test.
 				if deployment.HasDriver {
-					WaitForPMEMDriver(c, deployment.Namespace)
+					WaitForPMEMDriver(c, deployment.Name, deployment.Namespace)
```

Comment:
`deployment.Name` is incorrect here, the name is "pmem-csi".

Reply:
I also had to make changes to `WaitForPMEMDriver` (see  https://github.com/intel/pmem-csi/pull/667/commits/588e887141fdf48c8770528f6b9b4f9557267d95). To avoid conflicts, I cherry-picked your change and fixed it in https://github.com/intel/pmem-csi/pull/667/commits/9e23107ffec1d3590a40bb2e16b758436b2d1f56.

If you include that commit in your branch, we may be able to merge both branches without conflicts.

Reply:
I picked your commit.